src/amo/components/Collection/index.js

```diff
@@ -249,6 +250,11 @@ export class CollectionBase extends React.Component<Props> {
             editing={editing}
             loading={!collection || loading}
           />
+          {!loading && addons.length === 0 &&
+            <p className="Collection-placeholder">{ i18n.gettext(
```

Comment:
This formatting looks a bit odd. It might be better as:

```js
<p className="Collection-placeholder">
  {i18n.gettext('Search for extensions and themes to add to your collection.')}
</p>
```